packages/wallets/src/components/WalletCashier/WalletCashier.scss

```diff
@@ -4,4 +4,5 @@
     gap: 24px;
     justify-content: center;
     flex: 1;
+    background-color: #ffffff;
```

Comment:
We don't support dark theme on wallets?

Reply:
Currently we haven't cover dark mode yet